### ISSUE TITLE
Feedback 

### ISSUE THREAD

**Issue by DecentMakeover** [zylo117/Yet-Another-EfficientDet-Pytorch#194]
Hi 

Thanks for sharing your work, 

I am having a very hard time getting my dataset to train my labels looks this
![Screenshot from 2020-04-24 02-04-56](https://user-images.githubusercontent.com/25679895/80146861-29406800-85d0-11ea-83af-9fb513b05e7b.png)

i have tried training this with d4,d2,d0 but it does not converge at all, here is a sample when debug=True
![01a3e05175aa43beb8884098e9c60bf8](https://user-images.githubusercontent.com/25679895/80148384-9ce37480-85d2-11ea-91dc-794a6d02732e.jpg)

I have used another repo with the same annotation and  i am able to get good results in less than 3 epochs, Do you have any suggestions for me?

Thanks

**Comment by zylo117** [comment]
The result would be better if you train with `--head_only True`. Otherwise, you need large batchsize to train the backbone or it's very likely overfitting or underfitting.

Or you can remove the BiFPN from the list, so that training with `--head_only True` will also allow BiFPN to be trained while the backbone remains untouched.
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/master/train.py#L156

**Comment by DecentMakeover** [comment]
Actually the result is with --head_only True, let me try what you have suggested nonetheless and check if there are any improvements.
Thanks

**Comment by zylo117** [comment]
did you change mean and std, it matters if you are training on datasets from a complete different domain.
